npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater_spec.rb

```diff
@@ -1925,7 +1925,7 @@
       end
 
       context "with a dependency that can't be found" do
-        let(:manifest_fixture_name) { "non_existant_dependency.json" }
+        let(:manifest_fixture_name) { "nonexistent_dependency.json" }
```

Comment:
Can you also change the file name? `git mv npm_and_yarn/spec/fixtures/package_files/non_existant_dependency.json npm_and_yarn/spec/fixtures/package_files/nonexistant_dependency.json`